modules/mobile/docs/schemas/DiscoveryResponse.yml

```diff
@@ -0,0 +1,67 @@
+type: object
+required:
+  - data
+properties:
+  data:
+    type: object
+    required:
+      - id
+      - type
+      - attributes
+    properties:
+      id:
+        type: string
+        example: "1.0"
+        description: API Version number
+      type:
+        type: string
+        example: "discovery"
+      attributes:
+        type: object
+        required:
+          - endpoints
+          - oauthBaseUrl
+          - apiRootUrl
+          - webviews
+          - appAccess
+          - displayMessage
+        properties:
+          endpoints:
+            type: object
+            additionalProperties:
+              type: object
+            example:
+              appointments: {
+                url: "/v0/appointments",
+                verb: "GET"
```

Comment:
could we change `verb` to `method` as that's the terminology used in other parts of the API

Reply:
sure thing